### PR TITLE
First pass at correcting know issues with URLs in links and references.

### PR DESCRIPTION
**Pull request by oxelson**
- removed unidata downloads site index.jsp references
- removed references to my.unidata domain
- removed references to old package convention in unidata urls

### CODE REVIEW

File: it/src/test/data/thredds/catalog/InvCatalog-1.0.xml [hunk 1]
```diff
@@ -55,7 +55,7 @@
     <!-- example of documentation, attribute, and metadata elements -->
     <dataset name="Doc examples" dataType="Grid" serviceName="ACD" ID="hasProp">
       <property name="GoodThing" value="Where have you gone?"/>
-      <documentation xlink:href="http://my.unidata.ucar.edu/content/projects/THREDDS/currentwork.html">
```
Comment (lesserwhirls) [inline it/src/test/data/thredds/catalog/InvCatalog-1.0.xml:58]: I think I have some sticky notes with `my.unidata.ucar.edu`...good times!

**Comment by lesserwhirls** [comment]
Just the one downloads reference to fix (you can ignore the my.unidata comment). Thank you @oxelson!

File: docs/src/private/website/netcdf-java/reference/ToolsUI/ToolsUI.adoc [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
 === Download and run locally
 You can download `toolsUI.jar` from either the NetCDF-Java
-link:/downloads/netcdf/netcdf-java-4/index.jsp[downloads] or
+link:/downloads/netcdf/[downloads] or
```
[the remark this answers is not in the document]
Comment (oxelson) [reply docs/src/private/website/netcdf-java/reference/ToolsUI/ToolsUI.adoc]: @lesserwhirls 'tis fixed.

**Comment by lesserwhirls** [comment]
Thanks @oxelson!

**Pull request merged**